src/modules/Dashboard/components/CardImage/__tests__/CardImage.test.jsx

```diff
@@ -0,0 +1,130 @@
+import { configure, mount } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import * as React from "react";
+import { Button, Modal } from "react-bootstrap";
+import configureStore from "redux-mock-store";
+import { Provider } from "react-redux";
+import { cleanup } from "@testing-library/react";
+
+import CardImage from "../CardImage";
+import ConfirmDeleteModal from "../../ConfirmDeleteModal/ConfirmDeleteModal";
+import DownloadInfoModal from "../../DownloadInfoModal/DownloadInfoModal";
+import NotesModal from "../../NotesModal/NotesModal";
+
+configure({ adapter: new Adapter() });
+
+afterEach(cleanup);
+
+describe("CardImage", () => {
+    const mockStore = configureStore([]);
+    const deviceList = {
+        "rpi-4-b": {
+            name: "Raspberry Pi",
+            generation: 4,
+            model: "Model B",
+            distros: {
+                "raspbian-buster-armhf": {
+                    name: "Raspbian",
+                    codename: "Buster",
+                    version: 10,
+                    port: "armhf",
+                    packages_url: "https://packages.debian.org/buster/",
+                    build_types: ["classic"],
+                },
+            },
+        },
+    };
+    const image = (status = "Undefined") => ({
+        image_id: "4ed4c369-2fed-412e-b2e1-0d4d176e89fb",
```

Comment:
Let's use a pattern like `xxxxxxxx-xxxx-4xxx-xxxx-xxxxxxxxxxxx` to make it visible it's fake.